docs/relier-communication-protocols/sync-webchannel.md

```diff
@@ -0,0 +1,133 @@
+SYNC via WebChannels protocol
+=============================
+
+The preferred method for FxA to communicate with Firefox is with [WebChannels](https://developer.mozilla.org/en-US/docs/Mozilla/JavaScript_code_modules/WebChannel.jsm). WebChannels provide secure two-way communication between the browser and web content. Messages sent from content to the browser utilize `WebChannelMessageToChrome` custom events. Messages sent from the browser to content utilize `WebChannelMessageToContent` custom events.
+
+`FirefoxAccountsCommands` have been deprecated.
+
+## v1
```

Comment:
nit: I wonder if this might be confusing because we have fx-desktop-v1 and v2, so this is v1 of WebChannels but not the auth_broker